src/templates/post.js

```diff
@@ -187,7 +187,8 @@ const BlogPost = ({ data }) => {
           <div className={styles.authorContainer}>
             <div className={styles.authorInformation}>
               <img src={`images/${authorBlog.image}`} alt="" className={styles.authorImage} />
-              <Link className={classnames(styles.authorName, "text__paragraph__bold__black")} to={`/authors/${ _.kebabCase(authorBlog.author) }`}>{ authorBlog.displayName }</Link>
+              <Link className={classnames(styles.authorName, "text__paragraph__bold__black")} to={null}>{ authorBlog.displayName }</Link>
+              {/* The author name links to a page which is not created yet, the link taht you will need to use is: `authors/${ _.kebabCase(authorBlog.author) }` */}
```

Comment:
Why this comment here?

Reply:
Because, if the next programmer makes the page of authors, he or she has the correspondent link to that page in that comment.